app/_locales/en/messages.json

```diff
@@ -2091,6 +2111,9 @@
   "priorityFee": {
     "message": "Priority Fee"
   },
+  "priorityFeeProperCase": {
```

Comment:
So [apparently](https://www.computerhope.com/jargon/p/proper-case.htm#:~:text=Proper%20case%20is%20any%20text,of%20every%20word%20being%20capitalized.) `Proper Case` actually means the first letter of every word is capitalized, so the other `priorityFee` key should be called `priorityFeeProperCase`. TIL!

Reply:
fixed